db.go

```diff
@@ -530,6 +535,11 @@ func (db *DB) close() (err error) {
 	db.stopMemoryFlush()
 	db.stopCompactions()
 
+	// Delete the mem table if not already deleted.
+	if db.mt != nil {
```

Comment:
why do you need this?

Reply:
I used this for deleting the created memtable in ReadOnly mode. This is not needed as we are not creating the memtable in upfront now in read-only mode.